tests/lib/rules/padding-line-between-statements.js

```diff
@@ -0,0 +1,4445 @@
+/**
+ * @fileoverview Tests for padding-line-between-statements rule.
+ * @author Toru Nagashima
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require("../../../lib/rules/padding-line-between-statements");
+const RuleTester = require("../../../lib/testers/rule-tester");
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const MESSAGE_NEVER = "Unexpected blank lines before this statement.";
```

Comment:
Nitpick: This message refers to "blank lines", so it might be confusing when there is only one blank line before the statement. I think it would be slightly better to say "Unexpected blank line before this statement."